lfs/client.go

```diff
@@ -795,3 +820,12 @@ func setErrorHeaderContext(err error, prefix string, head http.Header) {
 		}
 	}
 }
+
+func PrintError(err error, format string, args ...interface{}) {
+	line := format
+	if len(args) > 0 {
+		line = fmt.Sprintf(format, args...)
+	}
+	fmt.Fprintln(ErrorWriter, line)
+	fmt.Fprintln(ErrorWriter, err.Error())
+}
```

Comment:
This is the type of error collecting thing that the `commands` package should take care of. The `lfs` package is more of a library with functions that should bubble errors up. The `commands` package is the CLI layer that prints and logs errors.